OPEN-SOURCE-DOCUMENTATION

```diff
@@ -49,3 +49,20 @@ Available under license:
     5. Products derived from this software may not be called
     "ColorBrewer", nor may "ColorBrewer" appear in their name, without
     prior written permission of Cynthia Brewer.
+
+* JavaScript/CSS Font Detector
```

Comment:
Not sure if we actually need to add this statement to the OPEN-SOURCE-DOCUMENTATION file.